compiler/rustc_middle/src/lint.rs

```diff
@@ -346,6 +361,13 @@ pub fn struct_lint_level<'s, 'd>(
                     );
                 }
             }
+            LintLevelSource::ForceWarn(_) => {
+                sess.diag_note_once(
+                    &mut err,
+                    DiagnosticMessageId::from(lint),
+                    "Warning forced by `force-warns` commandline option",
```

Comment:
```suggestion
                    "warning forced by `force-warns` commandline option",
```

our style guide requests lower case here